authors/wubwub/05-SegWit2x/hr_HR/final.md

```diff
@@ -0,0 +1,181 @@
+
+Kako se automobili na cesti gužvaju u 2 trake, tako transakcije ne mogu stati u blok i moraju čekati svoj red, no još jedan problem koji se ovdje javlja je taj što tko plati veću naknadu za transakciju, njegova transakcija ima i veći prioritet. To znači ne samo da se mreža uspori zbog puno transakcija, nego i naknade za transakcije rastu.
+
+Sad se vratimo 2 godine u prošlost kad je počela rasprava o skalabilnosti Bitcoina.
+Popularnost Bitcoina je skočila, a s time i stopa transakcija.
+Bitcoin nije mogao procesirati sve transakcije u očekivanom vremenu i gore navedeni scenarij je postao stvarnost.
+
+Drugi problem koji se u međuvremenu pojavio je bug po imenu [ASICBoost].
+
+ASICBoost omogućava veću efektivnost rudarenja. Manja potrošnja električne energije po GH/s (Gigahash/sekunda). Kineski proizvođač opreme za rudarenje, Bitmain imao je ugrađen ASICBoost u rudare koje su oni koristili, a time su zarađivali 30% Bitcoina više, što je otprilike do $100 milijuna godišnje.
+
+
+## Rješenje?
+
+Tu se javljaju problemi jer imamo suprotstavljena dva kampa: 
+
+1. Core developeri (programeri koji rade na Bitcoin protokolu)
+2. Kineski rudari
+
+### 1. Rješenje: 
+
+**Core developeri** žele implementirati Segregated Witness (SegWit u daljnjem tekstu), koje je kompleksnije rješenje, ali isplativije na duže staze. Također postavlja temelje za *Lightning network* koja će dodatno povećati skalabilnost s off-chain transakcijama. 
+
+### 2. Rješenje:
+
+Kineski rudari, s vlasnikom Bitmaina Jihan Wu na čelu, zagovarali su povećanje blokova, s 1mb na čak 8mb. To naravno nije dugoročno rješenje, jer kako se Bitcoin bude sve više i više koristio, tako ćemo nakon nekog vremena opet imati ovakve probleme.
+
+*No, stvarni razlog zašto su se kineski rudari bunili je taj što SegWit **onemogućava ASICBoost**, a time ostaju bez velike prednosti.*
+
+### 3. Rješenje:
+
+Prvo aktivirati SegWit, a kasnije povećati blokove.
+
+####  U čemu je problem?
+
+Tako drastično povećanje kapaciteta blokova bi negativno utjecalo na čvorove u mreži jer sada moraju pohraniti i obraditi 8 puta više podataka u istom vremenskom razdoblju, tako će inicijalna sinkronizacija trajati puno duže, trebat će više resursa da se čvor održi.
+S vremenom postoji mogućnost da zbog veličine blockchaina obični ljudi neće moći imati svoj čvor, nego će si to moći priuštiti samo veliki podatkovni centri i tvrtke, što će dovesti do centralizacije, a s time će Bitcoin postati korporativni talac. Onaj tko kontrolira čvorove, kontrolira i Bitcoin, zato je bitno da Bitcoin ostane pristupačan svima.
+
+Na primjer, ako dođe do centralizacije čvorova, oni koji ih kontroliraju mogu podiči limit s 21 milijun Bitcoina na koliko žele, mogu odbijati inače valjane transakcije, mogu mijenjati pravila protokola. To je vrlo opasno za Bitcoin, a i za ostatak kritpo prostora. 
```

Comment:
... kao što smo to objasnili u članku o Bitcoin Gold forku (link)

Reply:
Ne vidim da se igdje u tom postu spominje problematika centralizacije čvorova?

Reply:
Hmm true